integration_tests/correct_forwarder_snapshot.json

```diff
@@ -330,7 +330,7 @@
         "Tags": [
           {
             "Key": "dd_sls_plugin",
-            "Value": "v2.18.0"
+            "Value": "vX.XX.X"
```

Comment:
Do you think I should keep the major version normalized also? If I undid it then we would have to update the snapshots on each major version release.

Reply:
I don't think it matters, this seems fine to me